ansible/checker.yml

```diff
@@ -0,0 +1,48 @@
+# This playbook deploys a simple stand alone checker for SAP/HANS.
+
+- hosts: all
+
+  tasks:
+    - name: include load_facts
+      import_role:
+        name: load_facts
+      tags:
+        - always
+
+    - name: include 1.1.1 remediate
+      import_role:
+        name: '1.1.1'
+      tags:
+        - 1.1.1
+
+    - name: include 1.1.1.runtime remediate
+      import_role:
+        name: '1.1.1.runtime'
+      tags:
+        - 1.1.1.runtime
+
+    - name: include 1.2.1 remediate
+      import_role:
+        name: '1.2.1'
+      tags:
+        - 1.2.1
+
+    - name: include 1.3.5 remediate
+      import_role:
+        name: '1.3.5'
+      tags:
+        - 1.3.5
+
+    - name: set_global_results
+      set_fact:
+        global_results: "{{ (global_results |  default({})) }}{{ hostvars[item]['tests'] }}"
+      with_inventory_hostnames:
+        - all
```

Comment:
I think that this won't scale well. We are not going to have the host names under `all` most probably.
Anyway, we can leave this way by now